### PR TITLE
Tweak: Merge "Upgrade Swiper 8 library" feature to version [ED-14374]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: assets/dev/scss/frontend/conditionals/e-swiper.scss [hunk 1]
```diff
@@ -84,7 +83,7 @@
 		}
 
 		// `:where()` is used to decrease the specificity of the selectors.
-		:where( .swiper-container-horizontal ) {
+		:where( .swiper-horizontal ) {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/conditionals/e-swiper.scss:86]: @rami-elementor Have you tested this aspect of the code? Was there a bug in the previous version? Should it always have been `.swiper-container-horizontal, .swiper-horizontal`.

I believe we should test this manually.

Would you like me to test this?

File: assets/dev/scss/frontend/widgets/image-carousel.scss [hunk 2]
```diff
@@ -21,7 +20,7 @@
 }
 
 // Before the image-carousel layout is calculated by JS. //
-.elementor-image-carousel-wrapper:not(.swiper-container-initialized):not(.swiper-initialized) {
+.elementor-image-carousel-wrapper:not(.swiper-initialized):not(.swiper-initialized) {
```
Comment (hein-obox) [inline assets/dev/scss/frontend/widgets/image-carousel.scss:23]:
```suggestion
.elementor-image-carousel-wrapper:not(.swiper-initialized) {
```

File: assets/dev/scss/frontend/conditionals/e-swiper.scss (hunk 1 above)
Comment (rami-elementor) [reply assets/dev/scss/frontend/conditionals/e-swiper.scss:86]: Yes please.

Comment (hein-obox) [reply assets/dev/scss/frontend/conditionals/e-swiper.scss:86]: Your code seems to be correct.

**Review by hein-obox — APPROVED** [review]

**Review by arielk — APPROVED** [review]

**Pull request merged**